src/cljs/main/broadfcui/page/workspace/notebooks/notebooks.cljs

```diff
@@ -0,0 +1,330 @@
+(ns broadfcui.page.workspace.notebooks.notebooks
+  (:require
+   [broadfcui.common :as common]
+   [broadfcui.common.components :as comps]
+   [broadfcui.common.flex-utils :as flex]
+   [broadfcui.common.icons :as icons]
+   [broadfcui.common.input :as input]
+   [broadfcui.common.links :as links]
+   [broadfcui.common.modal :as modal]
+   [broadfcui.common.style :as style]
+   [broadfcui.common.table :refer [Table]]
+   [broadfcui.common.table.style :as table-style]
+   [broadfcui.common.table.utils :as table-utils]
+   [broadfcui.components.buttons :as buttons]
+   [broadfcui.components.collapse :refer [Collapse]]
+   [broadfcui.components.modals :as modals]
+   [broadfcui.config :as config]
+   [broadfcui.endpoints :as endpoints]
+   [broadfcui.page.workspace.monitor.common :as moncommon]
+   [broadfcui.utils :as utils]
+   [broadfcui.components.foundation-dropdown :as dropdown]
+   [clojure.set]
+   [clojure.string :as string]
+   [dmohs.react :as react]))
+
+
+(def machineTypes ["n1-standard-1"
```

Comment:
This is hardcoded in for now - in the future we'd like to possibly have a leo endpoint that returns the machine type choices